packages/webpack/template-webpack-plugin/src/LynxEncodePlugin.ts

```diff
@@ -199,7 +199,7 @@ export class LynxEncodePluginImpl {
                   }',globDynamicComponentEntry?globDynamicComponentEntry:'__Card__')`;
                 }
               }).join(',')
-            ),
+            ).filter(arr => arr.length).join(','),
```

Comment:
Instead of `.join`, can we split the `externalManifest` and `inlinedManifest` instead?


```js
            Object.keys(externalManifest).map(name =>
              `lynx.requireModuleAsync('${
                this.#formatJSName(name, publicPath)
              }')`
            ).join(','),
            ';module.exports=',
            Object.keys(inlinedManifest).map(name =>
              `lynx.requireModule('${
                this.#formatJSName(name, '/')
              }',globDynamicComponentEntry?globDynamicComponentEntry:'__Card__')`
            ).join(','),
            ';',
```

Reply:
Thanks for advice, I have replaced with this!